deploy/Makefile

```diff
@@ -1,21 +1,50 @@
 ENV_FILE=.env
 include $(ENV_FILE)
 
-DOCKER_REMOTE=ssh://$(ZEN_WEB_REMOTE)
-PROJECT_NAME=harmony
+DOCKER_HOST ?= ssh://$(ZEN_WEB_REMOTE)
+PROJECT_NAME ?= harmony-web
 
-web_configure:
+configure:
 	@scp nginx/nginx_vhost_default_location $(ZEN_WEB_REMOTE):${NGINX_DEFAULT_VHOST_CONFIG}
 	@scp instance_config.json $(ZEN_WEB_REMOTE):${INSTANCE_CONFIG}
 	@scp global_config.py $(ZEN_WEB_REMOTE):${GLOBAL_CONFIG}
 
-web_deploy:
-	@DOCKER_HOST=$(DOCKER_REMOTE) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml up -d --force-recreate
-	@DOCKER_HOST=$(DOCKER_REMOTE) docker system prune -af
+prune:
+	@DOCKER_HOST=$(DOCKER_HOST) docker system prune -af
 
-web_stop:
-	@DOCKER_HOST=$(DOCKER_REMOTE) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml down
-	@DOCKER_HOST=$(DOCKER_REMOTE) docker system prune -af
+up:
+	@DOCKER_HOST=$(DOCKER_HOST) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml up -d --force-recreate
+
+down:
+	@DOCKER_HOST=$(DOCKER_HOST) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml down
+
+logs:
+	@DOCKER_HOST=$(DOCKER_HOST) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml logs -f
+
+web_up:
+	@DOCKER_HOST=$(DOCKER_HOST) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml up -d --force-recreate web
+
+web_logs:
+	@DOCKER_HOST=$(DOCKER_HOST) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml logs web -f
+
+web_bash:
+	@DOCKER_HOST=$(DOCKER_HOST) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml run --rm web /bin/bash
 
 web_ssh:
 	ssh $(ZEN_WEB_REMOTE)
+
+hasura_logs:
+	@DOCKER_HOST=$(DOCKER_HOST) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml logs hasura -f
+
+postgres_up:
+	@DOCKER_HOST=$(DOCKER_HOST) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml --profile postgres up postgres -d --force-recreate
+
+postgres_psql:
+	@DOCKER_HOST=$(DOCKER_HOST) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml --profile postgres exec postgres psql -h localhost -U ${POSTGRES_USER}
+
+generate_connection_string:
+	@DOCKER_HOST=$(DOCKER_HOST) docker compose --project-name ${PROJECT_NAME} --env-file $(ENV_FILE) --file docker-compose.yml run web /bin/bash -c "ZEN_DB_LOG_ONLY=1 ./scripts/create_deployment_database.sh ${POSTGRES_DB_HOST} ${HARMONY_INSTANCE_DB_NAME} ${POSTGRES_USER}"
```

Comment:
`generate_connection_string` doesn't sound quite right here, rather rename to `setup_db....` something?

Reply:
Maybe `create_db_setup_script`?

It's spitting out sql that you have to copy and go run in in psql somewhere - so it's not actually doing the setup for you, just giving you the script.

Reply:
Yeah, that sounds better. It doesn't just generate the conn string 👍